From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Connor Tumbleson <connor.tumbleson@gmail.com>
Date: Sat, 8 Jul 2023 18:49:17 -0400
Subject: [PATCH 7/7] aapt1: allow duplicate bag items

---
 tools/aapt/ResourceTable.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tools/aapt/ResourceTable.cpp b/tools/aapt/ResourceTable.cpp
index b0d52dc3123c..307dd96453dd 100644
--- a/tools/aapt/ResourceTable.cpp
+++ b/tools/aapt/ResourceTable.cpp
@@ -3564,7 +3564,9 @@ status_t ResourceTable::Entry::addToBag(const SourcePos& sourcePos,
     // one an attr and one an id, with the same name.  Not something we
     // currently ever have to worry about.
     ssize_t origKey = mBag.indexOfKey(key);
-    if (origKey >= 0) {
+
+    // Apktool: Make statement false so we pack duplicate bag items if needed.
+    if (origKey >= 0 && 1 == 2) {
         if (!replace) {
             const Item& item(mBag.valueAt(origKey));
             sourcePos.error("Resource entry %s already has bag item %s.\n"
